src/main/java/seedu/address/commons/core/Commands.java

```diff
@@ -107,6 +108,10 @@
         for (int i = 0; i < ALL_COMMAND_WORDS.length; i++) {
             ALL_COMMANDS_AND_SHORT_MESSAGES.put(ALL_COMMAND_WORDS[i], ALL_SHORT_MESSAGE_USAGES[i]);
         }
+
+        for (int j = 0; j < ALL_COMMAND_ALIASES.length; j++) {
+            ALL_COMMANDS_AND_SHORT_MESSAGES.put(ALL_COMMAND_ALIASES[j], ALL_SHORT_MESSAGE_USAGES[j]);
+        }
```

Comment:
Is it possible to combine the two loops into one?

Reply:
Oops, of course it is. Will update in a minute, thanks!